src/components/ProjectCard/ProjectSearch.js

```diff
@@ -47,13 +48,23 @@ const ProjectSearch = props => {
             onChange={e => {
               setSearchString(e.target.value)
             }}
+            //added on key press for enter and then e.which for return
+            onKeyPress={ e => {
+              if (e.key === 'Enter' || e.which === 13) {
+                handleSearch(e, 1, searchString)
+              }
+            }
+          }
           />
           <Button
             label="Search"
+            variant="contained"
             type="submit"
             color="primary"
+            style={!searchString ? {background: '#c4c4c4'} : {background: '#1a61b0'}}
```

Comment:
Probably be better to break this out into classNames that trigger based on the condition vs the inline styling it's using now: So something like this `className={!searchString ? buttonColorChoice1 : buttonColorChoice2}`
Inline styling is best avoided if possible. 
But if this is the only way you could override MUI's styling, just let me know!